enterprise/backend/src/metabase_enterprise/advanced_config/util/urls.clj

```diff
@@ -0,0 +1,11 @@
+(ns metabase-enterprise.advanced-config.util.urls
+  (:require [metabase.models.setting :as setting :refer [defsetting]]
+            [metabase.public-settings :as public-settings]
+            [metabase.public-settings.premium-features :as premium-features]
+            [metabase.util.i18n :refer [deferred-tru]]))
+
+(defsetting pulse-url
+  (deferred-tru "By default \"Site Url\" is used in notification links, but can be overridden.")
+  :getter (fn []
+            (when (premium-features/enable-advanced-config?)
+              (or (setting/get-string :pulse-url) (public-settings/site-url)))))
```

Comment:
Agree with @noahmoss, this should be checking `(premium-features/hide-embed-branding?)` to see whether we have full app embedding enabled. `advanced-config` is an entirely different feature. 

The code should go in a namespace like `metabase-enterprise.embedding.utils` or something like that

Reply:
Done